guides/static_prerendering.md

```diff
@@ -60,6 +60,18 @@ module.exports = new Router().prerender(async () => {
 })
 ```
 
+## Prerendering with traffic data
+
+XDN allows to prerender based on site traffic. This will ensure that most popular pages are always prerendered.
+
+### Example: Prerender 100 most requested pages
+
+```js
+const { Router } = require('@xdn/core/router')
+
+module.exports = new Router().prerender([{ top: 100, headers: { foo: 'bar' } }])
```

Comment:
You need to expand on this.  What's the headers config mean?  What other configs are possible here?